ui_framework/src/components/form/form_row/form_row.js

```diff
@@ -0,0 +1,69 @@
+import React, {
```

Comment:
This worked and I applied it. The only negative is it won't register when the child has `isRequired` on the prop. I went ahead and removed is required from the child components.